contracts/Api3ToUsdReader.sol

```diff
@@ -0,0 +1,19 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@api3/airnode-protocol-v1/contracts/dapis/DapiReader.sol";
+import "./interfaces/IApi3ToUsdReader.sol";
+
+contract Api3ToUsdReader is DapiReader, IApi3ToUsdReader {
+    address public immutable reader;
```

Comment:
Why not using `Ownable` and put the `onlyOwner` modifier in read()? It seems to be the same with the benefit that ownership can be transferred, right?